js/src/settings/edit-contact-information/index.js

```diff
@@ -59,10 +47,7 @@ export default function EditContactInformation() {
 				backHref={ getSettingsUrl() }
 			/>
 			<div className="gla-settings">
-				<ContactInformation
-					view="settings"
-					onPhoneNumberChange={ setPhoneNumber }
-				/>
+				<ContactInformation view="settings" />
```

Comment:
:question: 
Shouldn't we still have `onPhoneNumberVerified={ setPhoneNumber }` still here? This way I'm not able to save the new & verified phone number at all.

Reply:
About this question, please refer to this https://github.com/woocommerce/google-listings-and-ads/pull/1019#discussion_r714460853.